packages/thumbprint-react/components/Modal/index.tsx

```diff
@@ -344,6 +318,14 @@ class ModalFooter extends React.Component {
                         return <div className={styles.modalFooterFluid}>{children}</div>;
                     }
 
+                    if (stickyFooterContainerRef === null) {
+                        return <div className={styles.modalFooterFluid}>{children}</div>;
+                    }
+
+                    if (stickyFooterContainerRef.current === null) {
+                        return <div className={styles.modalFooterFluid}>{children}</div>;
+                    }
```

Comment:
Combine into one conditional since the returns are the same?

Reply:
it seems like typescript isn't quite smart enough to be sure that the value of `.current` is not null if you combine them into a more complex boolean expression. I will refactor out the shared return value though.

Reply:
actually, i guess i had the wrong boolean expression before. got it to work now